composer.json

```diff
@@ -91,7 +91,9 @@
 
         "matthimatiker/opcache-bundle": "^0.7",
         "qandidate/toggle-bundle": "^1.1.0",
-        "google/recaptcha": "~1.1"
+        "google/recaptcha": "~1.1",
+        "phpoffice/phpword": "dev-master",
```

Comment:
We should specify a version instead of `dev-master` so that we won't pull in something unexpected later.